atomium/templates/page/page.component.inc

```diff
@@ -26,11 +26,13 @@ function atomium_preprocess_page(&$variables, $hook) {
     $variables['theme_hook_suggestions'][] = 'page__404';
   }
 
-  // Remove the "theres no content default yadi yada" from the frontpage.
-  unset($variables['page']['content']['system_main']['default_message']);
+  if (isset($variables['page']['content']['system_main'])) {
```

Comment:
You should use !empty, isset does not check if the array is empty. 
Also there is no certainty 'default_message' and '#theme_wrappers' are set because their parent is.